src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_OF_VALUES = 100;
+    private final Ball newBall = new Ball();
+    private final Random random = new Random();
+    private final ColorSupplier randomColor = new ColorSupplier();
+
+    public String getRandomBall() {
+        newBall.setColor(randomColor.getRandomColor());
+        newBall.setNumber(random.nextInt(NUMBER_OF_VALUES));
+        return newBall.toString();
```

Comment:
```suggestion
    private final Random random = new Random();
    private final ColorSupplier randomColor = new ColorSupplier();

    public Ball getRandomBall() {
        Ball newBall = new Ball();
        newBall.setColor(randomColor.getRandomColor());
        newBall.setNumber(random.nextInt(NUMBER_OF_VALUES));
        return newBall;
```
this method should return Ball object